metrics.py

```diff
@@ -78,6 +78,7 @@ def __str__(self):
     def __init__(self):
         super().__init__(requires_graph=False)
         self._is_directed = False
+        self._is_dyadic_independent = True
```

Comment:
Technically we need to override this field only when it's `False` (otherwise it will be inherited as `True`).
I'm not sure what's more readable...